lib/web_ui/lib/src/engine/text/ruler.dart

```diff
@@ -208,12 +209,12 @@ class TextHeightRuler {
     _dimensions.updateTextToSpace();
 
     _dimensions.appendToHost(host);
-    rulerHost.addElement(host);
+    rulerHost.addElement(host as html.HtmlElement);
```

Comment:
Huh... well this is a bit weird. I'm assuming this is going to change once `RulerHost` is updated to dom instead of HTML? If so, maybe just a comment/todo saying so?

Reply:
Yea, exactly. All of these casts will be removed in the end. I added a comment here, but this cast will become an error after `RulerHost` is migrated, so a TODO seems unnecessary.